visualization/include/pcl/visualization/impl/point_cloud_color_handlers.hpp

```diff
@@ -73,6 +73,47 @@ pcl::visualization::PointCloudColorHandlerCustom<PointT>::getColor (vtkSmartPoin
 }
 
 ///////////////////////////////////////////////////////////////////////////////////////////
+template <typename PointT> void
+pcl::visualization::PointCloudColorHandlerCustomIndividual<PointT>::setInputCloud (
+    const PointCloudConstPtr &cloud)
+{
+  PointCloudColorHandler<PointT>::setInputCloud (cloud);
+  
+  if (cloud_->points.size () == rgb_values_.size())
+    capable_ = true;
+  else
+    capable_ = false;
+}
+
+///////////////////////////////////////////////////////////////////////////////////////////
+template <typename PointT> bool
+pcl::visualization::PointCloudColorHandlerCustomIndividual<PointT>::getColor (vtkSmartPointer<vtkDataArray> &scalars) const
+{
+  if (!capable_ || !cloud_)
```

Comment:
Same comment here about the implicit bool cast on a `boost::shared_ptr`. Are you sure this yields false if the shared pointer is pointing to NULL?